test/integration/aws/cognito-user-pool.test.js

```diff
@@ -67,6 +67,7 @@ describe('AWS - Cognito User Pool Integration Test', function () {
     return deployService(serviceDir);
   });
 
+  // NOTE: do not refactor below into array function since we need `this` context
```

Comment:
Let's not add those comments.

Note that ESLInt will complain if we'd use full function without a reason. So if there's a full function in a code it most likely means there's a valid reason for that (no need to additionally comment on it)